pkg/helm/controller/controller.go

```diff
@@ -22,7 +22,9 @@ import (
 	"sync"
 	"time"
 
-	yaml "gopkg.in/yaml.v2"
+	// todo(camila): check the possibility to replace it for sigs.k8s.io/yaml since this dep has been added
+	// just because of this place
+	"gopkg.in/yaml.v3"
```

Comment:
I would say update this to `sigs.k8s.io/yaml` which uses `gopkg.in/yaml.v2` under the hood.

Reply:
It has not all methods used @estroz